src/notation/internal/notationinteraction.cpp

```diff
@@ -325,6 +325,15 @@ QList<Ms::Element*> NotationInteraction::hitElements(const PointF& p_in, float w
 
     PointF p = p_in - page->pos();
 
+    if (isTextEditingStarted()) {
+        auto editW = w * 2;
+        RectF hitRect(p.x() - editW, p.y() - editW, 2.0 * editW, 2.0 * editW);
+        if (m_textEditData.element->intersects(hitRect)) {
```

Comment:
doubling the "hit" tolerance might be excessive for higher values (it's a user preference), but certainly for the default value, it's surprisingly hard to ensure you can select text from the beginning or the end using the mouse. Even in Mu3 this is a bit of an issue.